src/styles/blocks/section-title.scss

```diff
@@ -0,0 +1,19 @@
+.section-title {
+  font-weight: 700;
```

Comment:
Consider using a variable for the font-weight to maintain consistency in design.